packages/contracts-bedrock/test/L1/OPContractsManager.t.sol

```diff
@@ -92,6 +91,18 @@ contract OPContractsManager_Harness is OPContractsManager {
     }
 }
 
+/// @notice Helper function to setup a prank for delegatecall
+/// @param _caller The address to prank as the caller.
+function prankDelegateCall(address _caller) {
+    // Calculated as address(uint160(uint256(keccak256("hevm cheat code"))))
+    Vm vm = Vm(0x7109709ECfa91a80626fF3989D68f67F5b1DD12D);
+    // Foundry fails with "cannot `prank` delegate call from an EOA" if empty
+    if (_caller.code.length == 0) {
+        vm.etch(_caller, hex"00");
+    }
+    vm.prank(_caller, true);
+}
```

Comment:
I created this as a free function here because I couldn't really find an appropriate place to put it, and it wasn't needed in ForkLive (the only other file that used DelegateCaller).

I could move it into CommonTest if preferred, but it doesn't quite match the nature of the existing functions there.